test/test.make

```diff
@@ -195,7 +195,7 @@ RUN_E2E = KUBECONFIG=`pwd`/_work/$(CLUSTER)/kube.config \
 		-ginkgo.randomizeAllSpecs=false \
 	        $(TEST_E2E_ARGS) \
                 -report-dir=$(TEST_E2E_REPORT_DIR)
-test_e2e: start $(RUN_TEST_DEPS)
+test_e2e: start $(RUN_TEST_DEPS) operator-generate-catalog
```

Comment:
Building the Operator SDK and then (to a lesser extend) building the catalog takes time. Should we perhaps reconsider the decision to not put the generated files into our repo, now that we do need them for every CI build?

OTOH, we then still need to rebuild them to catch broken PRs that do not properly update the committed files...

Let's keep it as-is (= not checked into repo).